grc/gui/Application.py

```diff
@@ -721,7 +725,7 @@ def flow_graph_update(fg=flow_graph):
                     print("could not terminate process: %d" % page.process.pid)
 
         elif action == Actions.PAGE_CHANGE:  # pass and run the global actions
-            pass
+            flow_graph_update()
```

Comment:
that's the right thing to do, here, I guess, but then the comment the line above is obsolete :)